### ISSUE TITLE
Function Call Tree filter duplicates ignores namespaces

### ISSUE THREAD

**Issue by sad-dev** [NationalSecurityAgency/ghidra#1682]
**Describe the bug**
If Cat::Walk and Dog::Walk call foo, the call tree window will only show one of them if filter duplicates is on

**Environment (please complete the following information):**
Ghidra 9.1.2

**Comment by dragonmacher** [comment]
I just tried this and it seems to be working for me.   Is there anything notable about Cat::Walk and Dog::Walk?
Is this in the incoming call tree for foo?

**Comment by sad-dev** [comment]
```
#include<iostream>
using namespace std;

void foo(const char * name) {

	cout << name << " walking\n";
	return;
}

class Cat  
{ 
    public: 
    void walk() { 
       foo("Cat");
    } 
    void fly() {
    	cout << "Cats must walk before they can fly\n";
    	walk();
    }
}; 


class Dog  
{ 
    public: 
    void walk() { 
       foo("Dog");
    } 
    void fly() {
    	cout << "Dogs must walk before they can fly\n";
    	walk();
    }
}; 

int main()
{
	Cat c;
	Dog d;
	c.fly();
	d.fly();
	return 0;
}
```

Results in the following incoming call tree for foo

![foo](https://user-images.githubusercontent.com/55233728/94504382-3aef0100-023b-11eb-8292-69d19feb6fc8.png)

**Comment by sad-dev** [comment]
As an aside, the demangler seems to yield weird results on the operator<< function,

``
import ghidra.app.util.demangler.DemanglerUti;
 DemanglerUtil.demangle("_ZStlsISt11char_traitsIcEERSt13basic_ostreamIcT_ES5_PKc");
``

results in 
`std::basic_ostream<char,std<char>> & std::operator<<<std--char_traits<char>>(std::basic_ostream<char,std::char_traits<char>> &,char const *)` in 9.12
and 
`std::basic_ostream<char,std::char_traits<char>> & std::operator<<<std::operator<<<std::char_traits<char>>(std::basic_ostream<char,std::char_traits<char>> &,char const *)` in 9.2

![cout](https://user-images.githubusercontent.com/55233728/94506233-7e4b6e80-023f-11eb-9686-f111faa5b4b2.png)

**Comment by dragonmacher** [comment]
> As an aside, the demangler seems to yield weird results on the operator<< function,

How did you know demangling goodies are a great way to get my attention?  :grin:

**Comment by dragonmacher** [comment]
Fixes in progress for both issues.

**Comment by sad-dev** [comment]
<< bug fixed in https://github.com/NationalSecurityAgency/ghidra/commit/bc4cf7de5fed5d490f8da8eb0bfcf38df892585d ?

**Comment by dragonmacher** [comment]
> << bug fixed in [bc4cf7d](https://github.com/NationalSecurityAgency/ghidra/commit/bc4cf7de5fed5d490f8da8eb0bfcf38df892585d) ?

Yes.  That commit fixed the demangler bug you provided above.
